pkg/morph/client/client.go

```diff
@@ -419,6 +420,30 @@ func (c *Client) IsValidScript(script []byte, signers []transaction.Signer) (res
 	return result.State == vmstate.Halt.String(), nil
 }
 
+// VotesFor returns a key the provided account has voted with its NEO
+// tokens for. Nil key with no error it returned if the account has no NEO.
+func (c *Client) VotesFor(addr util.Uint160) (*keys.PublicKey, error) {
+	c.switchLock.RLock()
+	defer c.switchLock.RUnlock()
+
+	if c.inactive {
+		return nil, ErrConnectionLost
+	}
+
+	neoReader := neo.New(c.rpcActor)
```

Comment:
Can have `nil` signers, doesn't really matter here (but a little less things to encode/decode for the request).

Reply:
you meant `New` -> `NewReader`? fixed

Reply:
Yeah, `NewReader` is faster and again, you can omit `c.rpcActor`.

Reply:
oh, you meant `neoReader := neo.NewReader(invoker.New(c.client, nil))`?

Reply:
the last push already has the `invoker` version. dont mind keeping it like that